src/MudBlazor/Services/MudGlobal.cs

```diff
@@ -147,7 +147,7 @@ public static class PopoverDefaults
         /// Only applies to components that use a <see cref="MudPopover"/> in conjunction with a <see cref="MudOverlay"/>
         /// to close the popover when a user clicks outside, such as <see cref="MudSelect{T}"/>.
         /// </remarks>
-        public static bool ModalOverlay { get; set; } = true;
+        public static bool ModalOverlay { get; set; } = false;
```

Comment:
```suggestion
        public static bool ModalOverlay { get; set; }
```

Not necessary to mark as false that is the default state.